src/com/company/Cricket_Game.java

```diff
@@ -0,0 +1,33 @@
+package com.company;
+
+import java.util.ArrayList;
+
+public class Cricket_Game {
+    public static void main(String[] args)
+    {
+        ArrayList<Team> teams = new ArrayList<Team>();
```

Comment:
dataType of Team should be List<Team>. List is an interface and ArrayList is its one implementation. We should declare dataType always as base class/interface, and initialize it with child class object.